tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -128,6 +128,9 @@ Create VCH - defaults
     Log To Console  Installer completed successfully: %{VCH-NAME}
 
     Run Regression Tests
+
+    ${sessions}=  Check vSphere Sessions  "vic-machine"
+    Should Be Equal As Integers  ${sessions}  0
```

Comment:
can you actually just add this to the regression tests keyword? Then you don't need it here and it will be run in a lot of other scenarios.